C/plugins/south/dht11/include/dht11.h

```diff
@@ -0,0 +1,28 @@
+#ifndef _DHT11_H
+#define _DHT11_H
+/*
+ * FogLAMP south service plugin
+ *
+ * Copyright (c) 2018 OSIsoft, LLC
+ *
+ * Released under the Apache 2.0 Licence
+ *
+ * Author: Amandeep Singh Arora
+ */
+#include <reading.h>
+
+#define MAXTIMINGS      85   /* was 85 */
```

Comment:
MAX_TIMINGS

in comment `was` ?

Reply:
Tried various values, during UT. Comment is not required anymore, removed it.